electrum/constants.py

```diff
@@ -78,7 +78,7 @@ class BitcoinMainnet(AbstractNet):
         'p2wsh':       0x02aa7ed3,  # Zpub
     }
     XPUB_HEADERS_INV = inv_dict(XPUB_HEADERS)
-    BIP44_COIN_TYPE = 34230
```

Comment:
AFAIK, 34230 is the mainnet port?

Reply:
this number is not mainport.
Used to be the same as the main port.

Reply:
it was just a random number?

Reply:
No.
This number is written this documents.
https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
i see. thanks to clarification!